diff --git a/browser/app/distribution/moz.build b/browser/app/distribution/moz.build
--- a/browser/app/distribution/moz.build
+++ b/browser/app/distribution/moz.build
@@ -4,7 +4,12 @@
 
 DIST_SUBDIR = ""
 
-if CONFIG["BUILT_BY_MOZILLA"]:
-    FINAL_TARGET_FILES.distribution += [
-        "distribution.ini",
-    ]
+FINAL_TARGET_FILES.distribution += [
+    "distribution.ini",
+    "policies.json",
+]
+
+FINAL_TARGET_FILES.distribution.extensions += [
+    "extensions/langpack-*@firefox.mozilla.org.xpi",
+    "extensions/uBlock0@raymondhill.net.xpi",
+]
