diff --git a/browser/components/customizableui/CustomizableUI.sys.mjs b/browser/components/customizableui/CustomizableUI.sys.mjs
--- a/browser/components/customizableui/CustomizableUI.sys.mjs
+++ b/browser/components/customizableui/CustomizableUI.sys.mjs
@@ -317,6 +317,11 @@ var CustomizableUIInternal = {
   initialize() {
     lazy.log.debug("Initializing");
 
+    if (!Services.prefs.getBoolPref("hilal.toolbar.initialized", false)) {
+      Services.prefs.clearUserPref("browser.uiCustomization.state");
+      Services.prefs.setBoolPref("hilal.toolbar.initialized", true);
+    }
+
     lazy.AddonManagerPrivate.databaseReady.then(async () => {
       lazy.AddonManager.addAddonListener(this);
 
